src/poetry/core/packages/dependency.py

```diff
@@ -258,6 +258,10 @@ def base_pep_508_name(self) -> str:
 
         return requirement
 
+    @property
+    def base_pep_508_name_resolved(self) -> str:
+        return self.base_pep_508_name
```

Comment:
However, this line is probably not tested now. 😉

For the sake of completeness, we should add a test with a non git dependency and pass `resolved=True`.

Reply:
Good point, I added this to one of the testcases in `test_dependency.py`